docs/azure/vscodeforweb.md

```diff
@@ -0,0 +1,193 @@
+
+9. `azd up` provisions the web app in creating the relevant Azure resources. Once done, you can see your application running in the browser by clicking the link provided in the terminal
+
+![Screenshot that shows azd up](images/vscodeforweb/fifteenth.png)
+
+10. Continue in VS Code Desktop or GitHub Desktop by clicking on “Continue on Desktop” in the bottom left corner. This button allows you to move your workspace to your local environment with one-click. If you have a dev container attached to an existing application, you have the choice to move using that container or to move local.
+
+![Screenshot that shows Continue On button](images/vscodeforweb/sixteenth.png)
+
+* [Spinning up a quick Azure Function app](https://learn.microsoft.com/en-us/azure/azure-functions/functions-develop-vs-code?tabs=node-v4%2Cpython-v2%2Cisolated-process%2Cquick-create&pivots=programming-language-python)
+
+The above quick start is applicable to vscode.dev/azure since vscode.dev/azure covers all of the prerequisites:
+- Python installed
+- Azure Functions Core Tools installed
+- Users will need to install these themselves:
+- Azure Functions extension
+
+* [Building and testing an AI agent using the Azure AI Toolkit](https://code.visualstudio.com/docs/intelligentapps/overview)
+
+Key features:
+- Model catalog with rich generative AI models sources (GitHub, ONNX, OpenAI, Anthropic, Google, ...)
+- Bring Your Own Models from remotely hosted model, or Ollama models that are running locally
+- Playground for model inference or testing via chat
+- Attachment support for multi-modal language models
+- Batch run prompts for selected AI models
+- Evaluate an AI model with a dataset for supported popular evaluators like F1 score, relevance, similarity, coherence, and more
+
+* [Rapid prototyping with VS Code extensions and Python](https://code.visualstudio.com/docs/python/python-quick-start)
+
+## 🔧 Limitations
+
+Although VS Code for the Web is almost at parity with VS Code Desktop, there are some limitations to the development environment:
+
+* No terminal access beyond Cloud Shell
+* Limited support for some native extensions or language features
+* No offline support
+
+## 🛠️ Troubleshooting
+
+If you encounter any issues with VS Code for the Web – Azure, please log an issue at in our [GitHub repository](https://github.com/microsoft/vscode-dev-azure)
+
+### Account type prompt issue
+If you see a prompt asking "What type of account did you use to start this tunnel?" then it's likely that you're on vscode.dev/azure and not insiders.vscode.dev/azure. Currently this product is only available on Insiders. To fix this, navigate to https://insiders.vscode.dev/azure.
+
+![Screenshot that shows Account Type](images/vscodeforweb/tbs1.png)
+
+### Connection issue
+If you're unable to connect to insiders.vscode.dev/azure, you can usually fix it by resetting your Azure Cloud Shell. This can be done from the Azure Portal.
```

Comment:
```suggestion
If you're unable to connect to <https://insiders.vscode.dev/azure>, you can usually fix it by resetting your Azure Cloud Shell. This can be done from the Azure Portal.
```